Automate-650 Remove shadow from text input fields Signed-Off-By: Tara…

tarablack01 opened the pull request:
… Black <tblack@chef.io>

### :nut_and_bolt: Description: What code changed, and why?
Some consistency clean up. Currently the filter box text input field has a shadow, whereas, most of our form input fields do not. In order to be consistent we will want to remove the box-shadow from both the client run and compliance filter text field.

### :chains: Related Resources
https://github.com/chef/automate/issues/650

### :+1: Definition of Done
There is no box shadow surrounding the text field.

### :athletic_shoe: How to Build and Test the Change
Run automate-ui

### :camera: Screenshots, if applicable
![Screen Shot 2019-07-29 at 8 30 31 PM](https://user-images.githubusercontent.com/4108100/62094052-54647980-b241-11e9-9e9e-5f00d5024779.png)
![Screen Shot 2019-07-29 at 8 30 43 PM](https://user-images.githubusercontent.com/4108100/62094053-54fd1000-b241-11e9-8aeb-b5d8de0ccd60.png)

susanev commented:
@tarablack01 here are a few more places we'd like to remove box shadows from, if anything turns out to be more complex let me know and ill make a separate issue.

client runs
- list page: chef client run status chart.. `chart-container`
- node detail page: node run summary box.. `node-run-summary`

compliance reports
- list page: all figures

tarablack01 commented:
> @tarablack01 here are a few more places we'd like to remove box shadows from, if anything turns out to be more complex let me know and ill make a separate issue.
> 
> **client runs**
> 
> * list page: chef client run status chart.. `chart-container`
> * node detail page: node run summary box.. `node-run-summary`
> 
> **compliance reports**
> 
> * list page: all figures

@susanev I've included these changes as well and have attached screenshots :)
Note: I added a border around node-run-summary.

<img width="1261" alt="Screen Shot 2019-07-31 at 4 43 17 AM" src="https://user-images.githubusercontent.com/4108100/62201899-e35eb800-b34d-11e9-9cff-169a83be554d.png">
<img width="642" alt="Screen Shot 2019-07-31 at 4 39 19 AM" src="https://user-images.githubusercontent.com/4108100/62201972-025d4a00-b34e-11e9-9933-e8b3800c99c9.png">
<img width="1254" alt="Screen Shot 2019-07-31 at 4 38 28 AM" src="https://user-images.githubusercontent.com/4108100/62201988-07ba9480-b34e-11e9-81be-abada9114c3e.png">

susanev commented:
thanks @tarablack01! ive got a question out to the ux team about the border and will follow-up once i hear back.

susanev commented:
@tarablack01 we'd like no border on `node-run-summary`, and if possible can you set the border-bottom on `chef-page-header` to zero for the node detail pages that would be rad. sorry for all the last minute asks.

tarablack01 commented:
@susanev I've implemented the requested changes, removed the border on `node-run-summary` and set the padding-bottom on `chef-page-header` to zero for the node detail pages
